app/Http/Controllers/NewsController.php

```diff
@@ -8,15 +8,79 @@
 use App\Libraries\CommentBundle;
 use App\Models\NewsPost;
 
+/**
+ * @group News
+ */
 class NewsController extends Controller
 {
-    public function __construct()
-    {
-        parent::__construct();
-
-        $this->middleware('require-scopes:public');
-    }
-
+    /**
+     * Get News Listing
+     *
+     * Returns a list of news posts and related metadata.
+     *
+     * ---
+     *
+     * ### Response Format
+     *
+     * Field                     | Type                    | Notes
+     * --------------------------|-------------------------|------
+     * news_posts                | [NewsPost](#newspost)[] | Includes `preview`.
+     * news_sidebar.current_year | number                  | Year of the first post's publish time, or current year if no posts returned.
+     * news_sidebar.news_posts   | [NewsPost](#newspost)[] | All posts published during `current_year`.
+     * news_sidebar.years        | number[]                | All years during which posts have been published.
+     * search.limit              | number                  | Clamped limit input.
+     * search.sort               | string                  | Always `published_desc`.
+     * cursor                    | [Cursor](#cursor)       | |
+     *
+     * <aside class="notice">
+     *   <a href="#newspost">NewsPost</a> collections queried by year will also include posts published in November and December of the previous year if the current date is the same year and before April.
+     * </aside>
```

Comment:
this is a lot to make sense of for a one-sentence notice but idk how else to word it 🤔